docs/tool_interface.rst

```diff
@@ -11,50 +11,60 @@ In order to describe the interface, a few definitions are required at first.
 
 .. _view-def:
 
-view
+View
```

Comment:
when we refer to them in the text, we'll get capital letters then, I don't know what I prefer ...

Reply:
Agree, it should not be capital letters when refered in text. It is weird to have non-caps h3 while h1 and h2 are caps but so it is.